agent/trend_watcher.py

```
@@ -1,683 +1,921 @@
+    def _load_cache(self) -> None:
+        if not self._cache_path.exists():
+            return
+        try:
+            data = json.loads(self._cache_path.read_text())
+            self._seen        = set(data.get("seen", []))
+            self._last_fetch  = float(data.get("last_fetch", 0.0))
+            self._fetch_count = int(data.get("fetch_count", 0))
+            for a in data.get("alerts", []):
+                try:
+                    self._alerts.append(TrendAlert(**a))
+                except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)